src/overzichtVeilingen.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <link href="https://cdn.jsdelivr.net/npm/daisyui@3.7.4/dist/full.css" rel="stylesheet" type="text/css" />
+  <script src="https://cdn.tailwindcss.com"></script>
+    <title>title</title>
+</head>
+<body>
+    <?php
+    include "connect.php";
+    include "functions/adminFunctions.php";
+    session_start();
+
+    $sql = "SELECT* FROM tblproducten"; 
+    $resultaat = $mysqli ->query($sql); 
+    
+    while ($data = $resultaat -> fetch_assoc()) {
+        $tijd = krijgResterendeTijd($data["eindtijd"]);
+        echo '<br><table>
+        <tr><td><img src="'.$data["foto"].'" alt="product heeft geen foto"></td></tr>
```

Comment:
wat als er geen foto is?

Reply:
alt="product heeft geen foto" staat toch in de code ;)

Reply:
Dat is niet in orde en moet veranderd worden in zoals een andere img dat zegt dat de image broken is. maar niet gewoon een tekstje want dat ziet er niet uit. 













































































dumbass